modules/network/arbitrum/index.ts

```diff
@@ -0,0 +1,43 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { data } from './data';
+import { workerJobs } from './workers';
+import { createServices } from './services';
```

Comment:
can we prefix these with the network name. I know its an opinion thing here, but exposing 10 versions of `data`, `workerJobs` and `createServices` to the global namespace is messy.

Reply:
that's done